src/sdk/model/vac/ba-job.ts

```diff
@@ -0,0 +1,192 @@
+import { BaJobJson } from './__json__/ba-job';
+
+/* istanbul ignore next: autogenerated */
+export class BaJob {
+
+  constructor(private _json: BaJobJson) {
+  }
+
+  /**
+   * Get id.
+   * @returns {string}
+   */
+  get id(): string {
+    return this._json.id;
+  }
+
+  /**
+   * Get name.
+   * @returns {string}
+   */
+  get name(): string {
+    return this._json.name;
+  }
+
+  /**
+   * Get status.
+   * @returns {string}
+   */
+  get status(): string {
+    return this._json.status;
+  }
+
+  /**
+   * Get type.
+   * @returns {string}
+   */
+  get type(): string {
+    return this._json.type;
+  }
+
+  /**
+   * Get last run.
+   * @returns {number}
+   */
+  get lastRun(): number {
```

Comment:
Date

Reply:
i forgot that we wanted to return Date object there... API side it is a timestamp so i didn't pay attention to it. Thank you for reporting, i'll update that.

Reply:
it doesn't look like you did update it?